CMakeCPack.cmake

```diff
@@ -0,0 +1,36 @@
+include(InstallRequiredSystemLibraries)
+ 
+set(CPACK_PACKAGE_NAME ${CMAKE_PROJECT_NAME})
+set(CPACK_PACKAGE_DESCRIPTION "A date and time library based on the C++11/14/17 <chrono> header ")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Zero runtime overhead utilities that make life easier when dealing with std::chrono data types")
+set(CPACK_PACKAGE_VENDOR "Howard Hinnant")
+set(CPACK_PACKAGE_CONTACT "howard.hinnant@gmail.com")
+set(CPACK_PACKAGE_VERSION_MAJOR "${MAJOR_VERSION}")
+set(CPACK_PACKAGE_VERSION_MINOR "${MINOR_VERSION}")
+set(CPACK_PACKAGE_VERSION_PATCH "${PATCH_VERSION}")
+set(CPACK_PACKAGE_FILE_NAME "${CMAKE_PROJECT_NAME}_${MAJOR_VERSION}.${MINOR_VERSION}.${PATCH_VERSION}")
+set(CPACK_SOURCE_PACKAGE_FILE_NAME "${CMAKE_PROJECT_NAME}_${MAJOR_VERSION}.${MINOR_VERSION}.${PATCH_VERSION}-src")
+
+SET(CPACK_SOURCE_IGNORE_FILES  "\\\\.#;/#;.*~")
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "/\\\\.git")
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "\\\\.bz2")
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "\\\\.gz")
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "/build/")    
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "/bin/")
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "/lib/")
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "Makefile")
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "Makefile\\\\..*")
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "CMakeCache.txt")
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "CMakeFiles")
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "cmake_install.cmake")
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "install_manifest.txt")
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "CPackConfig.cmake")
+LIST(APPEND CPACK_SOURCE_IGNORE_FILES "CPackSourceConfig.cmake")
+
+set(CPACK_GENERATOR "DEB" "TGZ" "ZIP")
```

Comment:
Can we add a RPM generator as well for non-Debian systems?

Reply:
I believe that simply adding "RPM" in line 30 is sufficient. I just did not have an rpm system at hand. Feel free to add it, test it and commit or PR into this branch.

@HowardHinnant FYI: the installer is still lacking the consideration of the IANA stuff, but i would like to address this in a separate branch after merge.